README.md

```diff
@@ -1,2 +1,110 @@
-# voice-aws-transcribe-php
-PHP script to transcribe voice to text using AWS Transcribe service
+# Amazon Transcribe with Nexmo Voice and AWS S3 using PHP
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://nexmo.dev/aws-nexmo-transcribe-php)
+
+PHP example script to be used as a callback by Nexmo Voice to prompt a caller for a message. Then, after the message is received, download the MP3 file to be uploaded to AWS S3 prior to creating a Amazon Transcribe job to convert the contents to text.
+
+## AWS Services
+
+For this example we will multiple AWS services, such as [AWS S3](https://aws.amazon.com/s3/) and [Amazon Transcribe](https://aws.amazon.com/transcribe/).
+
+In S3, create a bucket that will be used to store the MP3 files retrieved from Nexmo. Then make note of the bucket name for later inclusion to the `.env` file.
+
+Using the [AWS IAM](https://aws.amazon.com/iam/), add a new user and add the following permissions by importing the following JSON.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": "transcribe:*",
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+            "s3:PutObject",
+            "s3:GetObject",
+            "s3:DeleteObject"
+        ],
+            "Resource": "arn:aws:s3:::{bucket_name}/*"
+        }
+    ]
+}
+```
+
+
+## Running the App
+
+This sample app uses a `.env` file to provide the API key and URL to the IBM Tone Analyzer API.
+
+Copy the provided `.env.example` file to a new file called `.env`:
+
+```
+cp .env.example > .env
```

Comment:
This syntax isn't universal, so should just probably change it to `cp .env.example .env`

Reply:
Will add to new commit.